js/ControlTodoButton.js

```diff
@@ -0,0 +1,94 @@
+const viewAllListBtn = document.querySelector(".all"); 
+const viewTodoListBtn = document.querySelector(".active"); 
+const viewCompleteListBtn = document.querySelector(".completed"); 
+const filter = document.querySelector(".filters");
+
+export function initTodolistButton(){
+    filter.addEventListener("click", controlFilterButton);
+}
+
+function controlFilterButton({target}){
+    if(target.classList.contains('all')) chooseButton('all');
+    else if(target.classList.contains('active')) chooseButton('active');
+    else if(target.classList.contains('completed')) chooseButton('completed');
+}
+
+export function chooseButton(button){
+    if(button === 'all') viewAll();
+    else if(button === 'active') viewTodo();
+    else if(button === 'completed') viewDone();
+}
+
+
+function viewAll() {
+  // "전체보기" 버튼 클릭 시의 기능
+  const list = document.querySelectorAll(".todo-list>li");
+  for (let i = 0; i < list.length; i++) {
+    list[i].classList.add("selected");
+  }
```

Comment:
```suggestion
  list.map((li) => li.classList.add('selected'));
```

```for``` 문을 통해 전체 아이템을 순회하는 것도 좋지만 순회 메서드(map, filter, forEach)등을 적절하게 활용하신다면 더욱 보기 좋은 코드가 될거 같습니다!

Reply:
사용할 수 있는 부분에서 단순 반복문 대신 forEach 메서드를 사용하여 더 간결한 코드로 변경해보았습니다.
review#4 에 반영하여 수정하였습니다. 감사합니다!